src/librustc_trans/glue.rs

```diff
@@ -205,9 +204,15 @@ pub fn implement_drop_glue<'a, 'tcx>(ccx: &CrateContext<'a, 'tcx>, g: DropGlueKi
     // non-null, (and maybe we need to continue doing so), but we now
     // must definitely check for special bit-patterns corresponding to
     // the special dtor markings.
-    let v0 = get_param(llfn, 0);
     let t = g.ty();
 
+    let value = get_param(llfn, 0);
```

Comment:
Oh this meant you started using `value` below noo 😞, I'd rather you change it back.